docs/NooBaaNonContainerized/ConfigFileCustomizations.md

```diff
@@ -478,6 +478,34 @@ Warning: After setting this configuration, NooBaa will skip schema validations a
     3. systemctl restart noobaa
     ```
 
+### 33. Notification space monitor frequency flag -
+* <u>Key</u>: `NOTIFICATION_REQ_PER_SPACE_CHECK`
+* <u>Type</u>: Number
```

Comment:
Any limitation on this number? (only positive, not higher than...) you can write in the description.